Nukeeper.BitBucketLocal/BitbucketLocalRestClient.cs

```diff
@@ -111,9 +112,15 @@ public async Task<PullRequest> CreatePullRequest(PullRequest pullReq, string pro
             var requestJson = JsonConvert.SerializeObject(pullReq, Formatting.None, new JsonSerializerSettings {NullValueHandling = NullValueHandling.Ignore});
             var requestBody = new StringContent(requestJson, Encoding.UTF8, "application/json");
             
-            var response = await _client.PostAsync($@"projects/{projectName}/repos/{repositoryName}/pull-requests", requestBody);
+            var response = await _client.PostAsync($@"rest/api/1.0/projects/{projectName}/repos/{repositoryName}/pull-requests", requestBody);
 
             return await HandleResponse<PullRequest>(response, caller);
         }
+
+        public async Task<IEnumerable<AddReviewer>> GetBitBucketReviewers(string projectName, string repositoryName, [CallerMemberName] string caller = null)
+        {
+            var response = await GetResourceOrEmpty<List<Conditions>>($"rest/default-reviewers/1.0/projects/{projectName}/repos/{repositoryName}/conditions", caller);
+            return response.SelectMany(c => c.Reviewers).Select(usr => new AddReviewer() {User = usr} );
```

Comment:
So who will be the PR reviewer now? Is it the first in the reviewers list?

Reply:
All the default reviewers are added to the PR